### ISSUE TITLE
java.lang.NoClassDefFoundError: android/media/ThumbnailUtils on Android Studio

### ISSUE THREAD

**Issue by fcouceiro** [lopspower/CircularFillableLoaders#4]
I'm getting this error while previewing my layout in android studio when using your component. When I build the app and run it, the component doesn't show up and the app quits after 2/3 seconds.

Full android studio [stackstrace](http://jpst.it/E8ws).

Here is my xml definition:

```
<com.mikhaellopez.circularfillableloaders.CircularFillableLoaders
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:id="@+id/circularFillableLoaders"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    android:src="@drawable/ic_play"
    android:layout_alignParentTop="true"
    android:layout_centerHorizontal="true"
    app:border="true"
    app:border_width="12dp"
    app:progress="80"
    app:wave_amplitude="0.06"
    app:wave_color="#3f51b5" />
```

**Comment by lopspower** [comment]
Yes indeed, the use of `ThumbnailUtils` class does not allow to make preview. I am open if someone an idea to fix that.

**Comment by fcouceiro** [comment]
You could render a basic circle (as a placeholder) if the view is in edit mode, more [here](http://developer.android.com/reference/android/view/View.html#isInEditMode%28%29). That way one could still use the layout preview feature.

**Comment by lopspower** [comment]
I fix this bug with the last release 1.1.0:
`compile 'com.mikhaellopez:circularfillableloaders:1.1.0'`

**Comment by fcouceiro** [comment]
Thanks!
